module/TheWitcherTRPG.js

```diff
@@ -281,3 +282,46 @@ Handlebars.registerHelper("window", function (...props) {
 Handlebars.registerHelper("includes", function (csv, substr) {
     return csv.split(",").map(v => v.trim()).includes(substr);
 });
+
+
+/* -------------------------------------------- */
+/*  Combat Hooks                                */
+/* -------------------------------------------- */
+Hooks.on("updateCombat", async (combat, turn, data, options) => {
+    console.log("updateCombat hook triggered!");
```

Comment:
Can you move the content of the hook into a separate js file and call it as a method in this hook?

Reply:
Yes. What would be the best file to move it to? Is there already one present in the repo, do you mean effects.js or you mean a new one?

Reply:
You need to create one and name it like combatHooks. Its just so that the main file does not get bloated with all the necessary calls